www/web-proxy-useracl/src/opnsense/mvc/app/views/OPNsense/ProxyUserACL/index.volt

```diff
@@ -452,47 +179,10 @@ POSSIBILITY OF SUCH DAMAGE.
                 </td>
             </tr>
         </table>
-        {{ partial("layout_partials/base_dialog",['fields':formDialogMimes,'id':'DialogMimes','label':lang._('Edit mime types for black and white lists')]) }}
-    </div>
-    <div id="subtab_time" class="tab-pane fade in">
-        <h1 class="text-center">{{ lang._('Schedules') }}</h1>
-        <div class="alert alert-warning">{{ lang._('Note:') }} {{ lang._('Use this lists in ACL rules.') }}</div>
-        <div id="times">
-            <table id="times-content">
-                <tr>
-                    <td colspan="2">
-                        <table id="grid-times" class="table table-condensed table-hover table-striped table-responsive"
-                               data-editDialog="DialogTimes">
-                            <thead>
-                            <tr>
-                                <th data-column-id="Description" data-type="string" data-sortable="false"
-                                    data-visible="true">{{ lang._('Description') }}</th>
-                                <th data-column-id="commands" data-width="7em" data-formatter="commands"
-                                    data-sortable="false">{{ lang._('Commands') }}</th>
-                            </tr>
-                            </thead>
-                            <tbody>
-                            </tbody>
-                            <tfoot>
-                            <tr>
-                                <td></td>
-                                <td>
-                                    <button data-action="add" type="button" class="btn btn-xs btn-default"><span
-                                                class="fa fa-plus"></span></button>
-                                    <button data-action="deleteSelected" type="button"
-                                            class="btn btn-xs btn-default"><span
-                                                class="fa fa-trash-o"></span></button>
-                                </td>
-                            </tr>
-                            </tfoot>
-                        </table>
-                    </td>
-                </tr>
-            </table>
-        </div>
-        {{ partial("layout_partials/base_dialog",['fields':formDialogTimes,'id':'DialogTimes','label':lang._('Edit Schedules')]) }}
+        {{ partial("layout_partials/base_dialog",['fields':tab['formDialog'],'id':'Dialog' ~ tab['name'],'label':lang._('Edit ' ~ tab['title'] ~ ' for black and white lists')]) }}
```

Comment:
Translating this way probably does not go well. It should be a format string. I am not sure if it works here but in volt it is `|format(variables)`